cime_config/testdefs/ExpectedTestFails.xml

```diff
@@ -59,26 +59,6 @@
 
   <!-- fates test suite failures -->
 
-  <test name="ERS_Lm12.1x1_brazil.I2000Clm50FatesCruRsGs.cheyenne_intel.clm-FatesFireLightningPopDens">
```

Comment:
Thanks for removing these since they now should pass as of dev151. Looks like you ran the FATES baselines to show that's the case which is awesome.